nengo_loihi/tests/test_basic.py

```diff
@@ -0,0 +1,154 @@
+import nengo
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize("weights", [True, False])
+def test_ens_ens(allclose, plt, seed, Simulator, weights):
+    with nengo.Network(seed=seed) as model:
+        a = nengo.Ensemble(100, 1, label="a")
+        ap = nengo.Probe(a)
+        anp = nengo.Probe(a.neurons)
+        avp = nengo.Probe(a.neurons[:5], "voltage")
+
+        b = nengo.Ensemble(101, 1, label="b")
+        solver = nengo.solvers.LstsqL2(weights=weights)
+        nengo.Connection(a, b, function=lambda x: x + 0.5, solver=solver)
+        bp = nengo.Probe(b)
+        bnp = nengo.Probe(b.neurons)
+        bup = nengo.Probe(b.neurons[:5], "input")
+        bvp = nengo.Probe(b.neurons[:5], "voltage")
+
+        c = nengo.Ensemble(1, 1, label="c")
+        bc_conn = nengo.Connection(b, c)
+
+    with Simulator(model) as sim:
+        sim.run(1.0)
+
+    plt.figure()
+    output_filter = nengo.synapses.Alpha(0.02)
+    a = output_filter.filtfilt(sim.data[ap])
+    b = output_filter.filtfilt(sim.data[bp])
+    t = sim.trange()
+    plt.plot(t, a)
+    plt.plot(t, b)
+
+    assert anp in sim.data
+    assert avp in sim.data
+    assert allclose(a, 0.0, atol=0.03)
+
+    assert bup in sim.data
+    assert bvp in sim.data
+    firing_rate = np.sum(sim.data[bnp], axis=0) * sim.dt / sim.n_steps
```

Comment:
Technically, the firing rate in Hz is without multiplying by `sim.dt` (which can be written more simply as just `sim.data[bnp].mean(axis=0)`). Then you'll get an average firing rate that's in a more normal range (30-50 Hz).

Reply:
Yeah I made the name before multiplying by `sim.dt` (was doing that below before), then changed and didn't update the name. Feel free to change :+1: